### PR TITLE
SUSE fix file_groupowner_backup_etc_shadow

### PR DESCRIPTION
**Pull request by brett060102**
For sle systems the shadow back up file is group shadow
with a gid of 15.

#### Description:

- fix file_groupowner_backup_etc_shadow to work for SUSE

#### Rationale:

- current code does not match group ownership for sle systems.

### CODE REVIEW

File: linux_os/guide/system/permissions/files/permissions_important_account_files/file_groupowner_backup_etc_shadow/rule.yml [hunk 1]
```diff
@@ -45,4 +45,5 @@ template:
         filegid@ubuntu1604: '42'
         filegid@ubuntu1804: '42'
         filegid@ubuntu2004: '42'
+        filegid@sle15: '15'
```
Comment (vojtapolasek) [inline linux_os/guide/system/permissions/files/permissions_important_account_files/file_groupowner_backup_etc_shadow/rule.yml:48]: Could you please do it in the same way as here:
https://github.com/ComplianceAsCode/content/pull/8554

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Hello, just a nitpick based on a different PR. But looks good.

File: linux_os/guide/system/permissions/files/permissions_important_account_files/file_groupowner_backup_etc_shadow/rule.yml (hunk 1 above)
Comment (brett060102) [reply linux_os/guide/system/permissions/files/permissions_important_account_files/file_groupowner_backup_etc_shadow/rule.yml]: Done

**Review by vojtapolasek — APPROVED** [review]
Looks good now

**Pull request merged**